Source/Library/GameBoardWindow/GameBoardPanel.cpp

```diff
@@ -0,0 +1,45 @@
+#include <QGridLayout>
+#include "TicTacToe/Library/GameBoardWindow/GameBoardPanel.hpp"
+#include "TicTacToe/Library/GameBoardWindow/TokenTile.hpp"
+
+using namespace boost;
+using namespace boost::signals2;
+using namespace TicTacToe;
+
+GameBoardPanel::GameBoardPanel(const Board& board, QWidget* parent) :
+  QWidget(parent),
```

Comment:
Double indent line 10 and 11, we do this to make it clear that line 12 and below is separate from line 10 and 11.

    GameBoardPanel::GameBoardPanel(const Board& board, QWidget* parent)
        : QWidget(parent),
          m_board(&board) {
      setFixedSize(...);
    }

Reply:
Do we follow the google C++ style guide regarding constructor initializer lists (https://google.github.io/styleguide/cppguide.html#Constructor_Initializer_Lists)? I used a different coding style in the previous code.

Reply:
The example here is similar to what we use:

    MyClass::MyClass(int var)
        : some_var_(var),             // 4 space indent
          some_other_var_(var + 1) {  // lined up
      DoSomething();
    }

Reply:
Do I have to modify the code I previously completed? Like BadgePanel, GameModel, which I didn't follow the guide.

Reply:
For this starter project no. Just going forward use this style.